pkg/util/helper/patch.go

```diff
@@ -32,3 +50,40 @@ func GenMergePatch(originalObj interface{}, modifiedObj interface{}) ([]byte, er
 
 	return patchBytes, nil
 }
+
+// GenReplaceFieldJSONPatch returns the RFC6902 JSONPatch array as []byte, which is used to simply
+// add/replace/delete certain JSON **Object** field.
+func GenReplaceFieldJSONPatch(path string, originalFieldValue, newFieldValue interface{}) ([]byte, error) {
```

Comment:
Is there an implementation of this method in Kubernetes that we can call directly?

Reply:
Didn't see any jsonpatch helpers, only strategypatch and mergepatch helpers in `/staging/src/k8s.io/apimachinery/pkg/util/strategicpatch` and `/staging/src/k8s.io/apimachinery/pkg/util/jsonmergepatch`.

Reply:
Ok, thanks for your reply~